P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+            <w lemma="in" pos="APPRART" xml:id="m16.t11">im</w>
+            <w lemma="Stress" norm="Stress" pos="NN" xml:id="m16.t12">stress</w>
+            <w lemma="sein" pos="VAPP" xml:id="m16.t13">gewesen</w>
+            <w lemma="ich" pos="PPER" xml:id="m16.t14">ich</w>
+            <w lemma="Armer" pos="NN" xml:id="m16.t15">Armer</w>
+            <w lemma="lol" pos="AKW" xml:id="m16.t16">lol</w>
+          </post>
+        </egXML>
+      </p>
+    </div>
+    <div xml:id="CMCnames">
+      <head>Named Entities and Anonymisation</head>
+      <p>Named entities (NEs) may be marked up using <gi>name</gi> or the elements encoding different
+        subcategories of names as described in <ptr target="#ND"/> such as <gi>surname</gi> or
+          <gi>geogName</gi>, or <gi>rs</gi> for a general referencing string. In the following chat
+        example (adapted from <ptr target="#BIB_DCK"/>), nicknames are linked to a <gi>person</gi>
+        entry as shown in section <ref target="#CMCParticipants"/> via the <att>ref</att> attribute.
+          <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de" source="#BIB_DCK">
+          <post modality="written" generatedBy="system" rend="color:black" synch="#f2213001.t007"
+            type="standard" who="#f2213001.A04" xml:id="f2213001.m27.eg35">
+            <name ref="#f2213001.A04" type="NICK">
+              <w lemma="Konstanze" pos="NE" xml:id="f2213001.m27.t1">Konstanze</w>
+            </name>
+            <w lemma="versuchen" pos="VVPP">versucht</w>
+            <name ref="#f2213001.A03" type="NICK">
+              <w lemma="Nasenloch" pos="NN">nasenloch</w>
+            </name>
+            <w lemma="die" pos="ART">den</w>
+            <w lemma="Wunsch" pos="NN">wunsch</w>
+            <w lemma="zu" pos="PTKZU">zu</w>
+            <w lemma="erfüllen" pos="VVINF">erfüllen</w>
+            <!-- ... -->
+          </post>
+        </egXML>
+      </p>
+      <p>In the the following version of the same chat snippet, the text strings with the nicknames
+        have been replaced by category label strings for the purpose of anonymisation. <egXML
+          xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de" source="#BIB_DCK">
+          <post modality="written" generatedBy="system" rend="color:black" synch="#f2213001a.t007"
+            type="standard" who="#f2213001a.A04" xml:id="f2213001a.m27.eg35">
+            <name ref="#f2213001a.A04" type="NICK">
+              <w pos="NE" xml:id="f2213001a.m27.t1">
+                <gap reason="anonymisation" unit="token" quantity="1"/>
+                <supplied reason="anonymisation">[_FEMALE-PARTICIPANT-A04_]</supplied></w>
+            </name>
+            <w lemma="versuchen" pos="VVPP">versucht</w>
+            <name ref="#f2213001a.A03" type="NICK">
+              <w pos="NN">
+                <gap reason="anonymisation" unit="token" quantity="1"/>
+                <supplied reason="anonymisation">[_PARTICIPANT-A04_]</supplied></w>
```

Comment:
Anonymised names are confusing: FEMALE-PARTICIPANT-A04 and PARTICIPANT-A04? Either male/female/etc. or no information on gender at all but not "no mention and female" if you know what i mean.

Why are both "A04"? Shouldn't the number differ?